src/App.tsx

```diff
@@ -1,9 +1,25 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [movieName, setMovieName] = useState('');
```

Comment:
we check not only in movieName, but also in the description
```suggestion
  const [query, setQuery] = useState('');
```